libraries/AP_HAL_SITL/SITL_State.cpp

```diff
@@ -368,7 +368,7 @@ void SITL_State::_simulator_servos(struct sitl_input &input)
 
     input.wind.speed = wind_speed;
     input.wind.direction = wind_direction;
-    input.wind.turbulence = _sitl?_sitl->wind_turbulance:0;
+    input.wind.turbulence = _sitl?_sitl->wind_turbulance:0.0f;
```

Comment:
```suggestion
    input.wind.turbulence = _sitl->wind_turbulance;
```

Fixes the problem and now there's less code to have bugs in :-)

You can see at the top of this method we check for nullptr, and the same pointer is used unprotected elsewhere in here.

Reply:
Hey Peter,
Why did you remove ternary operator here, wasn't it there to protect from nullptr, also can you tell me where this sitl struct was originally defined.
Thanks

Reply:
> Hey Peter, Why did you remove ternary operator here, wasn't it there to protect from nullptr, also can you tell me where this sitl struct was originally defined. Thanks

I explained why it was safe to assume `_sitl` wasn't null in my earlier comment.

`_sitl` is initialised in `sitl_setup` from a singleton object.  The declaration of the variable is done in the base class for `SITL_State`, `SITL_State_Common.`